apps/web/app/(app)/environments/[environmentId]/settings/lookandfeel/components/UnifiedStyling.tsx

```diff
@@ -0,0 +1,428 @@
+            color={questionColor}
+            setColor={setQuestionColor}
+            description="Change the text color of the questions, descriptions and answer options."
+            className="max-w-full"
+            disabled={!unifiedStyling}
+          />
+
+          <ColorSelectorWithLabel
+            label="Input color"
+            color={inputColor}
+            setColor={setInputColor}
+            className="max-w-full"
+            description="Change the background color of the input fields"
+            disabled={!unifiedStyling}
+          />
+
+          <ColorSelectorWithLabel
+            label="Input border color"
+            color={inputBorderColor}
+            setColor={setInputBorderColor}
+            description="Change the border color of the input fields"
+            className="max-w-full"
+            disabled={!unifiedStyling}
+          />
+
+          <ColorSelectorWithLabel
+            label="Card background color"
+            color={cardBackgroundColor}
+            setColor={setCardBackgroundColor}
+            description="Change the background color of the card"
+            className="max-w-full"
+            disabled={!unifiedStyling}
+          />
+
+          <div className="flex flex-col gap-4">
+            <div className="flex items-center gap-6">
+              <Switch
+                checked={isHighlightBorderAllowed}
+                onCheckedChange={(value) => {
+                  setIsHighlightBorderAllowed(value);
+                }}
+                disabled={!unifiedStyling}
+              />
+              <div className="flex flex-col">
+                <h3 className="text-sm font-semibold text-slate-700">Add highlight border</h3>
+                <p className="text-xs text-slate-500">Add on outer border to your survey card</p>
+              </div>
+            </div>
+
+            {isHighlightBorderAllowed && (
+              <ColorPicker
+                color={highlightBorderColor}
+                onChange={setHighlightBorderColor}
+                containerClass="my-0"
+              />
+            )}
+          </div>
+
+          <div className="flex flex-col gap-4">
+            <div className="flex flex-col">
+              <h3 className="text-sm font-semibold text-slate-700">Roundness</h3>
+              <p className="text-xs text-slate-500">Change the border radius of the card and the inputs.</p>
+            </div>
+            <div className="flex flex-col justify-center rounded-lg border bg-slate-50 p-6">
+              <Slider
+                value={[roundness]}
+                max={22}
+                onValueChange={(value) => setRoundness(value[0])}
+                disabled={!unifiedStyling}
+              />
+            </div>
+          </div>
+        </div>
+
+        <div className="mt-8 flex items-center justify-end gap-2">
+          <Button variant="minimal" className="flex items-center gap-2" onClick={onReset}>
+            Reset
```

Comment:
We should add a confirm Modal here as it's too easy to just replace the look and feel of all surveys within the product with just one slightly misplaces click.